app/db/shopit.sql

```diff
@@ -0,0 +1,40 @@
+USE shopit;
+
+DROP TABLE IF EXISTS user;
+CREATE TABLE user
+(
+    id                  varchar(255) NOT NULl,
+    username            varchar(20) NOT NULL,    # main email address serve as username
```

Comment:
Shall we just name it "email" so it is the same name across tables?

Reply:
email is also confusing. maybe primary_email?